### ISSUE TITLE
PHP8.1 The Attribute syntax support

### ISSUE THREAD

**Issue by exploitfate** [newrelic/newrelic-php-agent#865]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

I've got `Segmentation fault (core dumped)` with [PHP 8 Attributes](https://php.watch/articles/php-attributes)

[NOTE]: # ( Describe the problem you're encountering. )
[TIP]:  # ( Do NOT give us access or passwords to your New Relic account or API keys! )

## Steps to Reproduce

```php
#[Route('/', name: 'health_check_index', methods: ['GET'])]
```

[NOTE]: # ( Please be as specific as possible. )

## Your Environment

Ubuntu 22.04
PHP8.1
newrelic-php5 10.19.0.9

[TIP]:  # ( Include as many relevant details about your environment as possible. )

**Comment by ZNeumann** [comment]
I am able to reproduce this issue both with and without the New Relic PHP Agent, by simply registering with PHP's Observer API (which our agent has used since v10.18). This means that it is likely an issue with PHP, not the agent. The good news is that I have been unable to reproduce with PHP 8.2. Can you try upgrading and see if the problem persists?

**Comment by ZNeumann** [comment]
closing, as this has been resolved in PHP 8.2
